src/fiskaltrust.ifPOS/v1/it/IITSSCD.cs

```diff
@@ -44,5 +44,35 @@ public interface IITSSCD
         /// </summary>
         [OperationContract(Name = "v1/NonFiscalReceipt")]
         Task<Response> NonFiscalReceiptAsync(NonFiscalRequest request);
+
```

Comment:
@TSchmiedlechner @steininger added a few more methods and tried to keep them as generic as possible. Basically we are separating cancellation from normal and receipts from invoices. 

There are a few things to consider IMO and I am not sure if it does make sense to have dedicated endpoints for invoices / receipts / ( maybe another one for logs?) or if we should leave that decision up to the SCU.